Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Blocklist_de Feed
+- Fixed an issue where tags field was not shown.
```

Comment:
please provide more detail - not shown where?

Reply:
Done